rest-demo/src/main/java/com/projectexample/restdemo/controller/CloudVendorController.java

```diff
@@ -10,7 +10,7 @@
 
 import java.util.List;
 import java.util.Optional;
-//Response Handling
+//Response Handling API
 @RestController
 @RequestMapping("/cloudvendor")
```

Comment:
There is API declaration with same URL mapping in file, com.projectexample.restdemo.service.CloudVendorService
I could not start the application. Please fix below error:
Caused by: java.lang.IllegalStateException: Ambiguous mapping. Cannot map 'cloudVendorController' method 
com.projectexample.restdemo.controller.CloudVendorController#DeleteCloudVendorDetails(String)
to **{DELETE [/cloudvendor]}: There is already 'cloudVendorAPIService' bean method**
com.projectexample.restdemo.controller.CloudVendorAPIService#DeleteCloudVendorDetails(String) mapped.